packages/db/src/project/test/integration.spec.ts

```diff
@@ -509,7 +520,7 @@ describe("Compilation", () => {
         );
         // @ts-ignore won't be undefined
         let bytecodeId: string = generateId("bytecodes", shimBytecodeObject);
-        let callBytecodeId = generateId("bytecodes", shimCallBytecodeObject);
+        //let callBytecodeId = generateId("bytecodes", shimCallBytecodeObject);
```

Comment:
Yup yup, it was unused in the test and going back through the history is unclear if it has/had value and I'm glad you called it out. 

It originates in: https://github.com/trufflesuite/truffle/commit/64fb1a77670ef3ca37ecb0e39c27c5dde8703490 from when the idField recipes/CASIDs are added. About 20 days later a change moves to inline mixins: https://github.com/trufflesuite/truffle/commit/f2bce1dc3809f2a6502d5ee01a38b17d5cfc8ba7

Some of the similar and previously used variables are removed (`sourceids`) but some are left behind (`bytecodeId`). I wasn't sure if it has significance in an 'if this throws the test should fail' and that was left in place for that reason since we kind of skirt the line between integration and e2e test.

I'll nuke these.

Reply:
Cool thanks!